### ISSUE TITLE
bower.json should reference only one copy of angular-ui-tree.js in main

### ISSUE THREAD

**Issue by doublemarked** [angular-ui-tree/angular-ui-tree#487]
Release 2.4.0 introduced `angular-ui-tree.min.js` to the list of files in bower.json's `main`. This is in addition to `angular-ui-tree.js`. Unfortunately, this causes tools like wiredep to inject both copies (minified and vanilla) of the library. 

I did a small comparison to other big packages and it does not seem standard to list both versions of the library. Can `angular-ui-tree.min.js` be removed from the list of `main` files in bower.json?

**Comment by Abrissirba** [comment]
+1

**Comment by ernestopye** [comment]
+1

**Comment by pedrolopix** [comment]
just create a pull request with, until is merged you can chance your bower.json 
from:
```json
"dependencies": {
  "angular-ui-tree": "~2.5.0"
}
```
to:
```json
"dependencies": {
  "pedrolopix-angular-ui-tree": "https://github.com/pedrolopix/angular-ui-tree.git"
}
```

**Comment by shantanubhadoria** [comment]
+1 please, It causes both js and min.js to be included in the html file when doing gulp inject!!

**Comment by Voles** [comment]
This has been fixed with #488. A new release will be out soon.

**Comment by jacobscarter** [comment]
Closing this issue due to inactivity. See #815 for more information.
Please create a new issue is this still occurs with the current version of the component.
